packages/ts-transformers/src/opaque-ref/call-kind.ts

```diff
@@ -0,0 +1,256 @@
+import ts from "typescript";
+
+const BUILDER_SYMBOL_NAMES = new Set([
```

Comment:
:+1: great pulling these out: will be valuable when changes are done to the commontools lib